conans/test/integration/toolchains/cmake/test_cmaketoolchain.py

```diff
@@ -352,6 +353,73 @@ def generate(self):
         assert "/path/to/builddir" in contents
 
 
+@pytest.fixture
+def lib_dir_setup():
+    client = TestClient()
+    client.save({"conanfile.py": GenConanfile().with_generator("CMakeToolchain")})
+    client.run("create . --name=onelib --version=1.0")
+    client.run("create . --name=twolib --version=1.0")
+    conanfile = textwrap.dedent("""
+            from conan import ConanFile
+
+            class Conan(ConanFile):
+                requires = "onelib/1.0", "twolib/1.0"
+
+            """)
+    client.save({"conanfile.py": conanfile})
+    client.run("create . --name=dep --version=1.0")
+
+    conanfile = (GenConanfile().with_requires("dep/1.0").with_generator("CMakeToolchain")
+                 .with_settings("os", "arch", "compiler", "build_type"))
+
+    client.save({"conanfile.py": conanfile})
+    client.run("install . -s build_type=Release")
+    return client
+
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="Only Windows")
+def test_lib_dirs_windows(lib_dir_setup):
```

Comment:
I think we probably need two tests, one for single config cmake generator, and one for multi-config, both of which can be tested on any platform, unless I'm missing something?